tests/conftest.py

```diff
@@ -189,3 +190,14 @@ def platform_account_fixture(django_db_setup, django_db_blocker, configure_setti
         )
 
         yield account_json, account_instance
+
+
+@pytest.fixture(autouse=True)
+def slow_down_tests(request):
```

Comment:
@arnav13081994 this isn't the right way to do it ... if you run into 429s the tests should know to retry

Reply:
I agree and was planning on adding `DJSTRIPEHTTPCLIENT` to the codebase anyway but this `wait` solution seems like a good enough fix (for now) just to be able to start adding tests using the `stripe api`. I can probably try to put together the `DJSTRIPEHTTPCLIENT` over the weekend but that would need to be very, very thoroughly tested as this would be a huge change because of potential duplicate objects getting created in production.

What do you think?

Reply:
I see. Is there anyway we can review Github Actions usage? It seems I do not have enough access.

Please check it [here](https://docs.github.com/en/billing/managing-billing-for-github-actions/viewing-your-github-actions-usage)

Reply:
We aren't limited in usage but I want to keep it to a minimum anyway. It's wasteful in energy and it increases iteration time on PRs.